.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.13.0
+    steps:
+      - run: echo "Placeholder step"
```

Comment:
@krise3 @harrisonywu Missing newline character at end of several files. Ending a file with a newline character is a common convention as it helps certain command-line tools display those files in a way that is easier to read. Linting with AirBnB's style guide should catch this.